client/web/compose/src/components/Common/RecordToolbar.vue

```diff
@@ -27,6 +27,29 @@
         </b-button>
       </div>
 
+      <b-button-group class="ml-auto">
+        <b-button
+          v-if="nextPrevNavigationOptions.prev"
```

Comment:
Make sure the whole section is not shown (button group), if the prop is missing. Also prop should be required: false, by default. So you can rely on it